src/components/CallToAction/ExtendedGetInstanceCallToAction.js

```diff
@@ -135,46 +94,33 @@ const ExtendedGetInstanceCallToAction = ({
           fullWidth
```

Comment:
This is currently showing `null` by default on the page. You might want to default it to empty string on the calling site